sql-plugin/src/main/spark340/scala/org/apache/spark/sql/rapids/shims/GpuCreateDataSourceTableAsSelectCommandShims.scala

```diff
@@ -67,10 +64,10 @@ case class GpuCreateDataSourceTableAsSelectCommand(
       }
 
       saveDataIntoTable(
-        sparkSession, table, table.storage.locationUri, child, SaveMode.Append, tableExists = true)
+        sparkSession, table, table.storage.locationUri, SaveMode.Append, tableExists = true)
     } else {
       table.storage.locationUri.foreach { p =>
-        GpuRunnableCommand.assertEmptyRootPath(p, mode, sparkSession.sessionState.newHadoopConf)
+        GpuDataWritingCommand.assertEmptyRootPath(p, mode, sparkSession.sessionState.newHadoopConf)
```

Comment:
This actually parallelizes more of what Spark 3.4 does anyways. So no need for this in GpuRunnableCommand.